node/cn/backend.go

```diff
@@ -312,6 +298,35 @@ func New(ctx *node.ServiceContext, config *Config) (*CN, error) {
 	return cn, nil
 }
 
+// setAcceptTxs sets AcceptTxs flag in 1CN case to receive tx propagation.
+func (s *CN) setAcceptTxs() error {
```

Comment:
How about this?
```suggestion
func (s *CN) setAcceptTxsFor1CN() error {
```

Reply:
`For1CN` sounds a little bit ambiguous. Let me merge this for now and let's change the name after if it still does not look proper.